controller/src/main/java/io/pravega/controller/task/Stream/TxnSweeper.java

```diff
@@ -86,7 +86,7 @@ public boolean isReady() {
                 RETRYABLE_PREDICATE, Integer.MAX_VALUE, executor);
         return hostsOwningTxns.thenComposeAsync(index -> {
             index.removeAll(activeHosts.get());
-            log.info("Failed hosts {} have orphaned tasks", index);
+            log.debug("Failed hosts {} have orphaned tasks", index);
```

Comment:
lets keep this at info as it indicates what we will be failing over

Reply:
ok